includes/managers/controls.php

```diff
@@ -941,6 +941,12 @@ public function add_page_transitions_controls( Controls_Stack $controls_stack, $
 			__( 'Page Transitions let you style entrance and exit animations between pages as well as display loader until your page assets load.', 'elementor' ),
 		];
 
+		// If the user has a Pro but the experiment is off.
```

Comment:
@StyleShit, just wonder is there no any other way without asking if the pro is loaded?

Reply:
How do you check if there is Pro without checking if there is Pro? 😅

Reply:
Maybe the side effects of the pro